recipes/libgfortran/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "3.0.0" %}
+
+package:
+  name: libgfortran
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true                    # [win]
+  always_include_files:
+    - lib/libgfortran.dylib     # [osx]
+    - lib/libgfortran.3.dylib   # [osx]
+    - lib/libgfortran.so        # [linux]
+    - lib/libgfortran.so.3      # [linux]
+    - lib/libgfortran.so.3.0.0  # [linux]
+
+    # Including libquadmath for the time
+    # being. This will need to be broken
+    # out in the long term.
+    - lib/libquadmath.dylib     # [osx]
+    - lib/libquadmath.0.dylib   # [osx]
+    - lib/libquadmath.so        # [linux]
+    - lib/libquadmath.so.0      # [linux]
+    - lib/libquadmath.so.0.0.0  # [linux]
+
+    # Including libgcc_s for the time
+    # being. This will need to be broken
+    # out in the long term.
+    - lib/libgcc_s.1.dylib         # [osx]
+    - lib/libgcc_s_ppc64.1.dylib   # [osx]
+    - lib/libgcc_s_x86_64.1.dylib  # [osx]
+    - lib/libgcc_s.so              # [linux]
+    - lib/libgcc_s.so.1            # [linux]
+
+requirements:
+  build:
+    - gcc 4.8.5
+    - cloog 0.18.0 10
```

Comment:
Please see this [comment]( https://github.com/conda-forge/isl-feedstock/issues/3#issuecomment-227037891 ).